actions/checks.create.tickets.yaml

```diff
@@ -0,0 +1,30 @@
+description: Creates tickets on atlassian
+enabled: true
+entry_point: src/openstack_check_actions.py
+name: checks.create.tickets
```

Comment:
Can we group these into `atlassian.create.tickets` instead of checks, as I can see other Atlassian based actions in the future